mappings/net/minecraft/world/gen/trunk/ForkingTrunkPlacer.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_5139 net/minecraft/world/gen/trunk/ForkingTrunkPlacer
```

Comment:
wouldn't `BranchedTrunkPlacer` be a better name? since we already refer to certain things as `Branched`

Reply:
This is from mojang's registration id

Reply:
god damn it